src/rpc/mining.cpp

```diff
@@ -815,7 +815,6 @@ UniValue estimatesmartfee(const JSONRPCRequest& request)
             "\n"
             "A negative value is returned if not enough transactions and blocks\n"
             "have been observed to make an estimate for any number of blocks.\n"
-            "However it will not return a value below the mempool reject fee.\n"
```

Comment:
Perhaps worth pointing out that the result may not be sufficient to be acceptable for the mempool?